DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
+Testing for the raw image from the DAQ to being written to the archive
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
+very small scale and it to test the components of a proof of concept
+that they all work together. ~\\[2\baselineskip]It will demonstrate
 
-It will demonstrate
 \begin{itemize}
-\item Data can be transferred into the lsst data facility (LDF).   
-\item The raw data images can be ingested by the DBB along with some metadata about the image and providence.  
-\item The data files  can then be retrieved by the LSP
+\tightlist
+\item
+  Data can be transferred into the lsst data facility (LDF). ~~
+\item
+  The raw data images can be ingested by the DBB along with some
+  metadata about the image and providence. ~
+\item
+  The data files ~can then be retrieved by the LSP
 \end{itemize}
 
-\subsection{Scope}
-\label{sect:scope}
+\subsection{Scope}\label{scope}
+
+The overall test plan for the LSST Data Management system is described
+in \citeds{LDM-503}.\\
+This document specifically refers to the milestone LDM-503-8b, which
+tests the raw image archiving for a CCOB device. ~
+
 
-The overall test plan for the LSST Data Management system is described in LDM-503.
-This document specifically refers to the milestone LDM-503-8b, which tests the raw image archiving for a CCOB device.  
 
 \subsection{System Overview}
 \label{sect:systemoverview}
 
-The raw image archiving system that is part of the LSST Data Management system is responsible for the data being stored into the archive environment at the LDF from various sources.  It will depend on what DAQ system (spectragraph, comcam, lsstcam, CCOB... others) as to where the data is located originally, and then how it will be sent to NCSA (LDF) .    The tests begin with data somewhere that needs to be stored at the archive enviornment at LDF and ingested into the DBB.   The LDM-503-5 milestone is limited to the raw image archive from a CCOB device, not all types of devices and is limited in scope.  
+The raw image archiving system that is part of the LSST Data Management
+system is responsible for the data being stored into the archive
+environment at the LDF from various sources. ~It will depend on what DAQ
+system (spectragraph, comcam, lsstcam, CCOB\ldots{} others) as to where
+the data is located originally, and then how it will be sent to NCSA
+(LDF) . ~ ~The tests begin with data somewhere that needs to be stored
+at the archive enviornment at LDF and ingested into the DBB. ~ The
+LDM-503-5 milestone is limited to the raw image archive from a CCOB
+device, not all types of devices and is limited in scope.
```

Comment:
LDM-503-05 is “alert distribution validation”; I don't think it's correct here.

Reply:
same as above.